examples/notebooks/beam-ml/run_inference_basic.ipynb

```diff
@@ -1362,6 +1308,408 @@
+            "\u001b[K     |████████████████████████████████| 151 kB 54.8 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 62 kB 1.4 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 124 kB 64.1 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 180 kB 38.9 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 83 kB 1.8 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 183 kB 52.2 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 255 kB 66.0 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 435 kB 52.7 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 173 kB 59.7 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 267 kB 55.2 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 267 kB 63.3 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 148 kB 57.6 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 235 kB 57.7 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 1.0 MB 51.5 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 119 kB 62.0 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 119 kB 53.5 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 118 kB 45.3 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 62.3 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 58.1 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 56.7 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 66.2 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 36.8 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 265 kB 56.4 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 148 kB 55.4 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 147 kB 65.9 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 4.6 MB 35.9 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 5.9 MB 43.6 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 1.1 MB 50.5 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 438 kB 49.5 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 1.7 MB 48.0 MB/s \n",
+            "\u001b[?25h  Building wheel for dill (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
+            "  Building wheel for google-apitools (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
+            "  Building wheel for docopt (setup.py) ... \u001b[?25l\u001b[?25hdone\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Authenticate\n",
```

Comment:
This cell is repeated

Reply:
I've removed all repeated cells and split notebooks into framework specific notebooks.